examples/tests/valid/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "jsii-code-samples": "1.7.0"
```

Comment:
Should we create an issue to manage a package like this in the monorepo? It should be small so the benefits of avoiding external deps and network lookups for tests would outweigh the cost of maintaining it.

Reply:
Sure, will do